ClangPowerTools/ClangPowerTools/clang-build.ps1

```diff
@@ -778,7 +804,7 @@ Function Run-ClangJobs([Parameter(Mandatory=$true)] $clangJobs)
     Push-Location $job.WorkingDirectory
 
     $callOutput = & $job.FilePath $job.ArgumentList.Split(' ') 2>&1 |`
-                  ForEach-Object { $_.ToString() }                  |`
+                  ForEach-Object { $_.ToString() -replace "^(.*):(\d+):(\d+): (.*)$",'$1($2): $4' } |`
```

Comment:
This needs more work.
We'll perform a better output processing from within the VS extension, for properly formatting messages for [Output] and [Error] tabs.